pkgs/by-name/ku/kubectl-get-all/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-get-all";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "stackitcloud";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-fVe9PY5MFX6Yc31dLi6C1bXZ39Em+Piq4EWoQce8srY=";
```

Comment:
```suggestion
    repo = "kubectl-get-all";
    tag = "v${version}";
    hash = "sha256-fVe9PY5MFX6Yc31dLi6C1bXZ39Em+Piq4EWoQce8srY=";
```

Reply:
I've set `tag` and `hash` now, but why should the repo get hard-coded?